kubestash/__init__.py

```diff
@@ -396,14 +491,87 @@ def cmd_push(args):
                                                                                              secret=args.secret))
 
 
-def cmd_daemon(args):
-    # https://boto3.readthedocs.io/en/latest/reference/services/dynamodb.html
-    # https://boto3.readthedocs.io/en/latest/reference/services/dynamodbstreams.html
-    # TODO: what if there are more than 100 streams?
+def cmd_pushall(args):
+    """Syncs a Credstash table with an entire cluster"""
 
-    # -f --force is implied by daemon
-    args.force = True
+    # Map of all namespaces to secrets
+    secretMap = {}
+
+    # Depending on whether or not we need to
+    if args.secretname:
```

Comment:
I'm sure a lot of this can be shortened with list comprehensions, but I wanna keep it for a latter update